tests/test_generate_mbz_toc.py

```diff
@@ -283,3 +285,66 @@ def test_toc_creation_page_and_lesson_together(
         assert lines[3] == \
             f'        * [{lesson_page_name}](./html/{html_filenames["L"]})'
         assert lines[4] == f'    * [{page_name}](./html/{html_filenames["P"]})'
+
+
+def test_toc_creation_page_and_lesson_together_csv(
+    mocker, tmp_path, mbz_builder, lesson_builder, page_builder
+):
+    lesson_html = "<p>Lesson Content</p>"
+    lesson_name = "Only Lesson"
+    lesson_page_name = "Lesson Page 1"
+    lesson = lesson_builder(
+            id=1,
+            name=lesson_name,
+            pages=[
```

Comment:
You can hit the code coverage gap in your test by adding a second page to this lesson.